db/migrate/20210608161920_add_content_id_to_subscriber_lists.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentIdToSubscriberLists < ActiveRecord::Migration[6.1]
+  def change
+    add_column :subscriber_lists, :content_id, :uuid
```

Comment:
I think we want an index on this field too, or matching content to these subscriber lists will be slow